spring-integration-core/src/test/java/org/springframework/integration/handler/MethodInvokingMessageProcessorTests.java

```diff
@@ -757,6 +869,11 @@ public String optionalAndRequiredHeader(@Header(required = false) String prop,
 			return prop + num;
 		}
 
+		public String optionalAndRequiredDottedHeader(@Header(name = "dot1.foo", required = false) String prop,
+				@Header(name = "dot2.baz", required = true) Integer num) {
```

Comment:
![redundant](https://cloud.githubusercontent.com/assets/1071590/22384053/9775bbb6-e49a-11e6-8d20-1b17b3b9439d.png)

And as you see IDEA shows unused methods.
So, that was strange with that getter before...